### ISSUE TITLE
Accessibility & tags

### ISSUE THREAD

**Issue by pi43r** [mntn-dev/t.js#15]
Hey,
I really like this library compared to others.
Just wanted to mention that it makes screen readers unusable, because the text gets separated into individual letters.
For some reason the screen reader reads the text normally if placed in a p-tag, but then the t-caret is placed below on the next line.

A workaround that I might try is to wait until everything is written and then join each separate letter back together.

But I think it would be nice if:
1) t.js could create semantic html
2) the caret would go inside child elements

**Comment by mntn-dev** [comment]
hi,

you'll need to set `<p>`'s CSS to `display:inline` (by default, it's [block-style](https://developer.mozilla.org/en/docs/Web/HTML/Block-level_elements)) in order to have/keep the caret well-placed, see [this demo](https://jsfiddle.net/s3uonek4/show/) at JSFiddle.

in general, to rely on Screen Readers for this kind of dynamic (typed) content should be avoided; maybe you can manage to exclude the whole to-type-text from getting parsed by the Reader (per tag attributes/API/whatever), and, instead, provide a 'static' copy of the text placed close, 'not-visible' to the type-content.
